### DISCUSSION TITLE
Diff shows no visual difference, but submission reports only 90.4% match - Daily Targets 17/07/2023

### DISCUSSION THREAD

**Discussion by I-NOZex** [cssbattle/forum#350] [Bugs reports]
![image](https://github.com/cssbattle/forum/assets/5805207/7fe67398-ee94-4b80-b0b7-074d5fdd9994)
![image](https://github.com/cssbattle/forum/assets/5805207/f5abbf5c-9d7f-4514-a436-4034d9ca0007)

https://github.com/cssbattle/forum/assets/5805207/263b3907-2fec-47a4-8a37-72b116e2e2d3

**Comment by chinchang** [comment]
Oh interesting! We would need your solution code to debug this further. You can DM us on [twitter](https://twitter.com/css_battle) or [discord](https://discord.gg/rvqvbcFM86).

**Comment by I-NOZex** [comment]
So I think I found the issue: browser default styles!
I develop my solution in Firefox, which doesn't add default margins to the _dt_, but aparently chromium-based browsers do.
My solution:
```
<p><dt><style>body{margin:0}dt,p{width:in;height:110;background:#060f55;border:40px solid #6592cf;border-bottom:0}dt{background:#6592cf;border:40px solid #060f55;border-top:0
```

**Reply by I-NOZex** [reply]
spacing the  _body_ selector with a wildcard would have made my solution 100% 🥲
